Bio/PDB/MMCIFParser.py

```diff
@@ -209,7 +228,11 @@ def _build_structure(self, structure_id):
             altloc = alt_list[i]
             if altloc in _unassigned:
                 altloc = " "
-            int_resseq = int(seq_id_list[i])
+            resseq = seq_id_list[i]
+            if resseq == ".":
+                # Non-existing residue ID
```

Comment:
I would issue a warning for each residue that is skipped, so that the user knows that this is the case.